vignettes/chapter0_GetStarted.Rmd

```diff
@@ -23,13 +23,15 @@ knitr::opts_chunk$set(echo = TRUE)
 This section will guide you through the process of configuring your work
 environment to initiate the use of GGIR. For this purpose, you will need
 at least one accelerometer file to be processed by GGIR, please feel
-free to download [this .gt3x file](https://www.dropbox.com/s/covfyj31sarfaw0/GGIR01.gt3x?dl=0) or [this .cwa file](https://biobank.ndph.ox.ac.uk/ukb/refer.cgi?id=131620)
+free to download [this .gt3x
+file](https://www.dropbox.com/s/covfyj31sarfaw0/GGIR01.gt3x?dl=0) or
+[this .cwa file](https://biobank.ndph.ox.ac.uk/ukb/refer.cgi?id=131620)
```

Comment:
Why are lines in many places in this PR split across multiple lines? It is not practical to have sentences split by line breaks everywhere, because when we revise the text we need to also revise all the line breaks. so, I think it is best to just keep sentences on a single line, where rendering in html will take care of how it fills up the page.

Reply:
This is unintentional and seems to be caused by using visual mode to revise the text. Will make sure to revise this.